aksharamukha/transliterate.py

```diff
@@ -517,6 +524,103 @@ def convert_default(src, tgt, txt, nativize = True, post_options = [], pre_optio
 
     return convert(src, tgt, txt, nativize, pre_options, post_options)
 
+'''tmp dir approach'''
+
+
+def convert_docx_files_tmp(src, tgt, docx_file, nativize, pre_options, post_options):
```

Comment:
Check with Willi, Should we keep 2 methodologies or remove this one and keep in-memory approach

Reply:
Good decision, I also like the in-memory better. If we get problems in production due to insufficient memory we can discuss and pull out your code for temp files again.